packages/next/build/webpack/loaders/next-flight-css-dev-loader.ts

```diff
@@ -4,13 +4,26 @@
  * inside a comment.
  */
 
-const NextServerCSSLoader = function (this: any, source: string | Buffer) {
+export function pitch(this: any) {
+  const content = this.fs.readFileSync(this.resource)
+  this.data.__checksum = (
+    typeof content === 'string' ? Buffer.from(content) : content
```

Comment:
It makes sense to just use the original content instead of a checksum because for dev speed is more important here.